server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -161,7 +166,10 @@ public StatusDTO status() {
             .setModeReason(modeChangeReason != null ? modeChangeReason.toString() : null)
             .setModeChangeTime(modeChange.getChangeTime())
             .setManagerCapabilities(caps)
-            .setTimeUTC(new Date());
+            .setTimeUTC(new Date())
+            .setResource(resource)
```

Comment:
These three additions introduce an interesting problem. In the vast majority of expected cases, it will have no value and will always be null, but we still want them to be available as necessary.

In this case, I would argue the solution here is to add a subclass of StatusDTO that provides the three additional fields. If we're operating with keycloak enabled, the subclass should be used and those fields set. Otherwise, we default to the "standard" StatusDTO that does not provide those fields.

The fields in the DTO should probably also be clearer. Maybe keycloakRealm or some such.